### ISSUE TITLE
Phone numbers with exts in Facility health services do not follow VA guidelines

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12717]
## Describe the defect
Currently in Facility health services, a phone number with an ext looks like this:
`<a aria-label="8 0 0. 8 5 7. 3 3 8 4" href="tel:800-857-3384p45543">800-857-3384 x 45543</a>`
However, this does not follow [VA content style guidelines](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers). Some updates are needed:

- For the text label, use ext. at the end instead of just x: 800-857-3384, ext. 45543.
- The extension should be included in the aria-label `aria-label="8 0 0. 8 5 7. 3 3 8 4. extension. 4 5 5 4 3."`
- For the href:
  - +1 should be included (this is for all phone numbers, not just ones with exts)
  - instead of a "p" for the ext, a comma should be used
  - `href="tel:+18008573384,45543'

### Additional Context

- There is a [<va-telephone> component](https://design.va.gov/components/telephone) that is available that will format these correctly. My guess is that this was originally done long before the component existed, as this is relatively new.  However, if we are going to update these now, it could be a possible, and preferable, option rather than custom code.
- Also, currently the "p" for the ext does not seem to be working/supported on all phones.  This was found and tested by Martha Wilkes (which is how it was brought to my attention) so this does seem to be something we might want to see if we can get on the roadmap sooner rather than later if at all possible

## To Reproduce
1. Go to [Jamaica Plain page](https://www.va.gov/boston-health-care/locations/jamaica-plain-va-medical-center/)
3. Click on the accordion for Blind and low vision rehabilitation accordion to expand it
4. Scroll down to the phone number and notice that there is an extension
5. Inspect the code to see that number doesn't match the VA standards, including the href has a "p" before the ext and that the aria-label does not include the ext.
![image](https://user-images.githubusercontent.com/106678594/220769363-85cc799e-c824-4c10-98fe-f72e73fb0d03.png)

## AC / Expected behavior

Note that the following AC address fixing the front end:
- [ ] Update href of all phone numbers (not just ones with exts) to include +1
- [ ] Update href of phone numbers to have a comma to introduce exts instead of a "p"
- [ ] Update aria-label to include ext when applicable
- [ ] I expect that non clinical services are addressed IF shared templates exist that would result in non clinical services also being updated 

Note the following AC address implementing a new component which would fix the front end display when used:
- [ ] I expect the front end phone component is implemented (its unclear if the existing components are self contained) 

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by CherylEvans** [comment]
This ticket was recreated in the va.gov-team repo Ticket # [64848](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64848)
